packages/core/src/config/storage.ts

```diff
@@ -57,6 +58,10 @@ export class Storage {
     return path.join(Storage.getGlobalGeminiDir(), TMP_DIR_NAME);
   }
 
+  static getGlobalBinDir(): string {
+    return path.join(Storage.getGlobalGeminiDir(), BIN_DIR_NAME);
```

Comment:
lets reuse the existing tmp directory rather than adding an extra directory as we recently did some work to reduce the number of subdirectories we create to avoid user confusion.

Reply:
This SGTM, however, it's worth pointing out that `temp` folders tend to be fairly safe to delete, i.e. logs. Whereas, these binaries aren't really "temporary" in that sense, i.e. because if temporary files are cleared at runtime, and our binaries are in there, then the SearchTool will become useless. Of course, it would be safe to delete these binaries at other points in time, we will just re-download them(another sort of odd behavior for something "temporary", it's more like a cache at that point).

Still, I fully understand the desire for consistency, so I'm happy to make this change if these binaries meet the project definition of "temporary." Thoughts?